v0/destinations/am/config.js

```diff
@@ -107,7 +107,7 @@ const Event = {
   }
 };
 
-const BASE_URL = "https://api.amplitude.com";
+const BASE_URL = "https://api2.amplitude.com";
 const BASE_URL_EU = "https://api.eu.amplitude.com";
```

Comment:
EU region endpoint is not updated right?

Reply:
yes @Gauravudia , eu region endpoint is same as previous